### PR TITLE
Long options & spaces vs tabs

### PR DESCRIPTION
**Pull request by mrc0mmand**
So, this cherry-pick contains one potentially controversial patch, which replaces tabs with spaces. Since this comes from a personal preference (as it makes formatting of certain code parts much easier in my opinion) I'd ask @matusmarhefka to chime in, since he's the original author, if this change is okay with him or if he wants to stick with tabs. If not, that's fine, I'll drop it and rework the patches that depend on it (hence why it's in this PR in the first place).

### CODE REVIEW

File: src/rand.c [hunk 1]
```diff
@@ -49,25 +49,23 @@ static unsigned short df_gdouf;
...
-	NULL
+        "AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
+        "%s%s%s%s%s%s%s%s%s%n%s%n%n%n%s%s%s%s%s%s%s%s%s%s%s%s%s%s%s%s%s%s",
+        "%n%n%n%n%n%n%n%n%n%n%n%n%n%n%n%n%n%n%n%n%n%n%n%n%n%n%n%n%n%n%n%n"
+        "%n%n%n%n%n%n%n%n%n%n%n%n%n%n%n%n%n%n%n%n%n%n%n%n%n%n%n%n%n%n%n%n",
+        "bomb(){ bomb|bomb & }; bomb",
+        ":1.285",
+        "org.freedesktop.foo",
+        "/org/freedesktop/foo",
+        "rm -rf /",
```
Comment (matusmarhefka) [inline src/rand.c:65]: I would remove these `rm` commands as they might be quite destructive in case of some bug.

File: src/dfuzzer.c [hunk 2]
```diff
@@ -1075,144 +1070,118 @@ void df_print_process_info(int pid)
...
-	}
-
-	if (iflg && !oflg) {
-		df_fail("Error: Object path is required if interface specified!\n"
-				"See -h for help.\n");
-		exit(1);
-	}
+        int c = 0;
+
+        static const struct option options[] = {
```
Comment (matusmarhefka) [inline src/dfuzzer.c:1075]: This is a nice addition, can you please also update the man page at https://github.com/matusmarhefka/dfuzzer/blob/master/man/dfuzzer.1 ?

**Review by matusmarhefka — COMMENTED** [review]
I have no problem with switching to spaces instead of tabs.

File: src/rand.c (hunk 1 above)
Comment (mrc0mmand) [reply src/rand.c]: Makes sense, dropped.

File: src/dfuzzer.c (hunk 2 above)
Comment (mrc0mmand) [reply src/dfuzzer.c:1075]: Definitely, updated!

**Pull request merged**